src/NHibernate.Test/Linq/QueryCacheableTests.cs

```diff
@@ -76,21 +76,23 @@ public void QueryIsCacheable3()
 		public void QueryIsCacheableWithRegion()
 		{
 			Sfi.Statistics.Clear();
-			Sfi.QueryCache.Clear();
+			Sfi.EvictQueries();
```

Comment:
`Sfi.QueryCache.Clear()` cleans only default cache region. Named cache regions remain untouched. So need to clear them separatly